lib/edge/targeting.ts

```diff
@@ -43,11 +41,6 @@ async function Targeting(config: Required<OptableConfig>, id: string): Promise<T
   return response;
 }
 
-function LmpidFromCache(config: Required<OptableConfig>): string | null {
-  const ls = new LocalStorage(config);
-  return ls.getLmpid();
-}
-
 function TargetingFromCache(config: Required<OptableConfig>): TargetingResponse | null {
```

Comment:
We might want to add an option to set some eids as GAM Secure Signals automatically